python/gtsam/examples/Pose2ISAM2Example.py

```diff
@@ -0,0 +1,155 @@
+"""
+GTSAM Copyright 2010-2018, Georgia Tech Research Corporation,
+Atlanta, Georgia 30332-0415
+All Rights Reserved
+Authors: Frank Dellaert, et al. (see THANKS for the full author list)
+
+See LICENSE for the license information
+
+Pose SLAM example using iSAM2 in the 2D plane.
+Author: Jerred Chen, Yusuf Ali
+Modelled after:
+    - VisualISAM2Example by: Duy-Nguyen Ta (C++), Frank Dellaert (Python)
+    - Pose2SLAMExample by: Alex Cunningham (C++), Kevin Deng & Frank Dellaert (Python)
+"""
+
+from __future__ import print_function
+import math
+import numpy as np
+from numpy.random import multivariate_normal as mult_gauss
```

Comment:
don't import functions, ever. and definitely don't change their names. Just use `np.random.multivariate_normal`